bwa.wdl

```diff
@@ -39,12 +39,14 @@ task Mem {
     String picardCommand = if (defined(bwaIndex.altIndex)) then sortSamCommand + " | " + setNmMdAndUqTagsCommand
     else sortSamCommand
 
+    String readgroupArg = if (defined(readgroup)) then "-R '" + readgroup + "'" else ""
```

Comment:
This should work with the original code right? Is this bug reported?

Reply:
not sure, it's an error when there is a suffix instead of a prefix. Maybe this is intended?